pkg/build/api/validation/validation_test.go

```diff
@@ -319,6 +321,40 @@ func TestValidateBuildParameters(t *testing.T) {
 	}
 }
 
+func TestValidateBuildParametersSuccess(t *testing.T) {
```

Comment:
not that there's anything wrong with this test, but what does it have to do with this change?